lms/services/exceptions.py

```diff
@@ -12,23 +12,23 @@ class ExternalRequestError(Exception):
     :type details: JSON-serializable dict or ``None``
     """
 
-    def __init__(self, explanation=None, response=None, details=None):
+    def __init__(
+        self, explanation="External request failed", response=None, details=None
```

Comment:
Changed this to the more human-friendly `"External request failed"` rather than `"ExternalRequestError"` because it's actually meant to be shown to users in the frontend's error dialogs.